include/mbedtls/bignum.h

```diff
@@ -129,6 +129,7 @@
         #endif /* !MBEDTLS_HAVE_INT64 */
 typedef  int64_t mbedtls_mpi_sint;
 typedef uint64_t mbedtls_mpi_uint;
+#define MBEDTLS_MPI_UINT_MAX                18446744073709551615UL
```

Comment:
Does this have to be a preprocessor constant (and not just a compile-time constant)? If not we can use a single definition:
```
#define MBEDTLS_MPI_UINT_MAX ((mbedtls_mpi_uint) -1)
```
`-1` cast to an unsigned integer type is guaranteed to be the largest value of that type. The only downside is that you can't use it in a preprocessor conditional since the preprocessor doesn't know about types.

Reply:
I used it in a preprocessor conditional so I unfortunately won't be able to use that.